sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Internals/Statsbeat/Statsbeat.cs

```diff
@@ -38,8 +51,86 @@ static Statsbeat()
 
         private static Measurement<int> GetAttachStatsBeat()
         {
-            // TODO: Add additional properties required for statbeat
-            return new(1, new("cikey", Customer_Ikey), new("language", "dotnet"));
+            if (s_resourceProvider == null)
```

Comment:
how frequently this method get called, 15 mins or 24 hours? If it is 15 mins, it's better to check and set the value at startup.

Reply:
this is 24 hrs